core/templates/dev/head/css/oppia.css

```diff
@@ -2135,10 +2137,21 @@ div#ng-curtain {
   font-size: 21px;
   font-weight: 300;
   height: 40px;
-  margin-left: 6px;
   margin-top: 8px;
 }
-.oppia-navbar-profile-dropdown-toggle {
+.oppia-logo-wide {
+  margin-left: 25px;
+}
+.oppia-logo-small {
+  margin-left: 6px;
+}
+
+.nav > li > a.oppia-navbar-tab {
+  font-size: 1em;
+  padding-top: 17px;
+}
+.oppia-navbar-profile-dropdown-toggle,
```

Comment:
Here and below: it seems to me like you might as well change .oppia-navbar-profile-* throughout the code to .oppia-navbar-*, since there doesn't seem to be much real difference between the two. In particular, I _think_ .oppia-navbar-profile-dropdown and .oppia-navbar-dropdown work exactly the same (right)?